instrumentation/jetty-httpclient/jetty-httpclient-9.2/library/src/main/java/io/opentelemetry/instrumentation/jetty/httpclient/v9_2/internal/JettyClientHttpAttributesExtractor.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.jetty.httpclient.v9_2.internal;
+
+import io.opentelemetry.instrumentation.api.instrumenter.http.HttpAttributesExtractor;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.eclipse.jetty.client.api.Request;
+import org.eclipse.jetty.client.api.Response;
+import org.eclipse.jetty.http.HttpField;
+import org.eclipse.jetty.http.HttpHeader;
+import org.eclipse.jetty.http.HttpVersion;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+final class JettyClientHttpAttributesExtractor extends HttpAttributesExtractor<Request, Response> {
+  private static final Logger LOG =
+      LoggerFactory.getLogger(JettyClientHttpAttributesExtractor.class);
+
+  @Override
+  @Nullable
+  protected String method(Request request) {
+    return request.getMethod();
+  }
+
+  @Override
+  @Nullable
+  protected String url(Request request) {
+    return request.getURI().toString();
+  }
+
+  @Override
+  @Nullable
+  protected String target(Request request) {
+    String queryString = request.getQuery();
+    return queryString != null ? request.getPath() + "?" + queryString : request.getPath();
+  }
+
+  @Override
+  @Nullable
+  protected String host(Request request) {
+    return request != null ? request.getHost() : null;
```

Comment:
Thanks to @mateuszrzeszutek's finding on passing `null` request on exception, are we able to remove the null checks on request now? `request` isn't nullable in the instrumenter API (we don't have nullness checking in our build yet but would eventually) so we don't expect it to be null, I think we just accidentally passed it before.

Reply:
done, thanks